examples/blog/ecommerce/src/interfaces/index.ts

```diff
@@ -2,7 +2,7 @@ export interface IProduct {
     id: string;
     title: string;
     description: string;
-    image: any;
+    image: any; // eslint-disable-line
```

Comment:
```suggestion
    image?: null | { url: string };
```

Lets add a type for it based on the usage 😅

Reply:
done